pkg/services/ssosettings/ssosettingsimpl/service.go

```diff
@@ -212,7 +211,33 @@ func (s *Service) Delete(ctx context.Context, provider string) error {
 	if !s.isProviderConfigurable(provider) {
 		return ssosettings.ErrNotConfigurable
 	}
-	return s.store.Delete(ctx, provider)
+
+	social, ok := s.reloadables[provider]
+	if !ok {
+		return ssosettings.ErrInvalidProvider.Errorf("provider %s not found in reloadables", provider)
+	}
+
+	err := s.store.Delete(ctx, provider)
+	if err != nil {
+		return err
+	}
+
+	currentSettings, err := s.GetForProvider(ctx, provider)
+	if err != nil {
+		return err
```

Comment:
Not sure if we want to return an error here because at this point the SSO settings have been successfully deleted. wdyt?